pages/pipelines/command_step.md.erb

```diff
@@ -241,8 +241,36 @@ steps:
 ```
 {: codeblock-file="pipeline.yml"}
 
+If you retry a job, the information about the failed job(s) remains, and a new job is created. The history of retried jobs is preserved and immutable.
 
-## Automatic retry attributes
+<%= image "retry-time-date.png", width: 2304/2, height: 698/2, alt: "You can view how and when a job was retried" %>
```

Comment:
Yeah, the hidden job is confusing because it it's not related to retries as far as I can tell?

Reply:
Mmmhm, the way I understand it - the "hidden" stuff - _is_ the history of retries - this is how I see it in 
![1597540813-1iorwy1rwxsxk9rxp-kyhtq](https://user-images.githubusercontent.com/27306876/167635308-125e72c2-9883-472d-9695-da1d68e03736.gif)
the gif that introduced this feature in the original blog post -

Reply:
OH I SEE